layouts/jobs/list.html

```diff
@@ -1,15 +1,53 @@
 {{ define "main" }}
-<section class="grid grid-cols-3 grid-rows-4">
+<section class="grid grid-cols-3 grid-rows-4 gap-3">
 	<div class="col-span-2">
 		{{ .Content }}
 	</div>
 	<div class="row-span-2">
 		<h1>Character image</h1>
 		<img src="" />
 	</div>
-	<div class="row-span-2"><h1>Job Fundamentals</h1></div>
-	<div class="row-span-2"><h1>Basics</h1></div>
+	<div class="row-span-2">
+		<h1>Job Fundamentals</h1>
+		<div>
+			{{ with .Site.GetPage "leveling-guide" }}
+				<h3>Leveling guide</h3>
+				<p>Patch: {{ .Params.patch }}</p>
+				<p>Updated: {{ time.Format "2 Jan, 2006" .Params.lastmod }}</p>
+			{{ end }}
+		</div>
+		<div>
+			{{ with .Site.GetPage "skills-overview" }}
+				<h3>Skills Overview</h3>
+				<p>Patch: {{ .Params.patch }}</p>
+				<p>Updated: {{ time.Format "2 Jan, 2006" .Params.lastmod }}</p>
+			{{ end }}
+		</div>
+	</div>
+	<div class="row-span-2">
+		<div>
+			{{ with .Site.GetPage "basic-guide" }}
+				<h3>Basic Guide (LV 90)</h3>
+				<p>Patch: {{ .Params.patch }}</p>
+				<p>Updated: {{ time.Format "2 Jan, 2006" .Params.lastmod }}</p>
+			{{ end }}
+		</div>
+		<div>
+			{{ with .Site.GetPage "openers" }}
+				<h3>Openers (LV 90)</h3>
```

Comment:
Small, but you could make the opener level a parameter instead just for future extendibility.